src/sagemaker/workflow/pipeline.py

```diff
@@ -744,6 +738,20 @@ def upsert(
         raise RuntimeError("Immutable Pipelines cannot be updated")
 
 
+@attr.s
+class PipelineList:
+    """PipelineList class to encapsulate a list of Pipeline objects
+
+    Attributes:
+        pipeline_list (List[Pipeline]): A list of Pipeline objects
+        next_token (str): If the result of the previous call was truncated, a token that can be used to retrieve
+            the next set of Pipeline objects
+    """
+
+    pipeline_list: Sequence[Pipeline] = attr.ib(factory=list)
```

Comment:
As the step class name is _PipelineList, let's make this attribute name to be pipelines

It's similar to StepCollection and step: https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/workflow/step_collections.py#L39